server/src/main/java/io/deephaven/server/plugin/python/PyObjectRefCountedNode.java

```diff
@@ -0,0 +1,40 @@
+package io.deephaven.server.plugin.python;
```

Comment:
A point from code review with Jianfeng over slack - should this go in a different package/module? It needs to be in a project that can see both jpy's PyObject and engine-updategraph's LivenessArtifact, and I wasn't immediately happy with any of the choices I saw...

Reply:
I think this is a valid question.  I would have looked for it in Integrations, but I think this is more of a @jmao-denver question.